packages/vats/src/lib-board.js

```diff
@@ -32,40 +31,50 @@ const calcCrc = num => {
 /**
  * Create a board to post things on.
  *
- * @param {number} [seed=0]
+ * @param {number | bigint} [firstSequence=0]
+ * @param {string} [prefix]
  * @returns {Board}
  */
-function makeBoard(seed = 0) {
+function makeBoard(firstSequence = 0, prefix = 'board0') {
   const idToVal = makeStore('boardId');
   const valToId = makeStore('value');
-  const sparseInts = generateSparseInts(seed);
+  let lastSequence = BigInt(firstSequence);
 
   /** @type {Board} */
   const board = Far('Board', {
     // Add if not already present
     getId: value => {
       if (!valToId.has(value)) {
-        // Retry until we have a unique id.
-        let id;
-        do {
-          const num = /** @type {number} */ (sparseInts.next().value);
-          id = `${num}${calcCrc(num)}`;
-        } while (idToVal.has(id));
+        lastSequence += 1n;
+        const seq = `${lastSequence}`;
```

Comment:
This seems more like a type conversion than a string template:
```suggestion
        const seq = String(lastSequence);
```
(As a template, reader has to see whether there are any other changes going on.)

Reply:
I'm following this as a convention established by other parts of our code.

But I agree, the `String` call is clearer in at least this case.

Reply:
kk, changing this is out of scope